src/calculator/test_calculator.py

```diff
@@ -1,13 +1,24 @@
-from . import add, subtract, multiply
+"""Test module for calculator operations."""
 
-def test_add():
-    assert add(2, 3) == 5
+from . import add, multiply, subtract
+
+# Test constants
+TEST_SUM = 5
+TEST_DIFF = 2
+TEST_PRODUCT = 6
+TEST_NEG_PRODUCT = -5
```

Comment:
is this because `ruff` tells you no magic numbers?
it is less readable this way than inline.